Matching game header component #20

ssinikoglu opened the pull request:
Please review. This PR consists of Image and words matching game header component. I will also make other PR's for image word matching game components and container. Test fails because the component is not meaningful alone.

ibrahimaljarad commented on src/components/MatchingGameHeader/index.jsx:
```diff
@@ -0,0 +1,17 @@
+    </>
+  );
+};
+
```
hey sule can you delete th extra newlines

ssinikoglu replied:
Thanks for reviewing Ibrahim. Ihave made the change that you have requested. PTAL

ssinikoglu commented:
PR is approved. PTAL @Ammar-64

Ammar-64 commented:
> PR is approved. PTAL @Ammar-64 

@ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again

ssinikoglu commented:
> > PR is approved. PTAL @Ammar-64
> 
> @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again

Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

ssinikoglu commented:
> > > PR is approved. PTAL @Ammar-64
> > 
> > 
> > @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again
> 
> Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

I have updated and add Arabic translations

> > > PR is approved. PTAL @Ammar-64
> > 
> > 
> > @ssinikoglu can you please update the branch, pull the dev branch, merge it with yours and push again
> 
> Thanks @Ammar-64 ! I have just updated the branch and pushed to Github.

I have updated branch also added Arabic translations @Ammar-64, PTAL.
